etc/concepts.tsv

```diff
@@ -209,20 +209,20 @@ Quiver	Quiver	995	QUIVER
 Greegree	Greegree	2804	AMULET
 Sacrifice	Sacrifice	1103	SACRIFICE
 Hell	Hell	878	HELL
-Waterfresh	Water (fresh)		
+Waterfresh	Water (fresh)	948	WATER
```

Comment:
Maybe leave it unmapped? In the Concepticon, WATER refers very broadly to any H₂O liquid, including that from rivers/body tissues, etc.